### PR TITLE
Math: Rename to MathUtils.

### LINKED ISSUE #9399: THREE.Math is now _Math, probably need a better name that doesn't conflict with built-in Math.
##### Description of the problem

We should probably figure out a new name for this class as without the THREE namespace, it conflicts with the built in JavaScript Math class.

See here:

https://github.com/mrdoob/three.js/blob/dev/src/math/Math.js
##### Three.js version
- [x] Dev
- [ ] r79
- [ ] ...
##### Browser
- [x] All of them
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer
##### OS
- [x] All of them
- [ ] Windows
- [ ] Linux
- [ ] Android
- [ ] IOS

### PR DESCRIPTION
**Pull request by Mugen87**
Fixed #9399.

Because of `export { MathUtils as Math };` in `Three.Legacy.js`, the change is backwards-compatible for now.

### CODE REVIEW

**Comment by Mugen87** [comment]
Travis is failing right now since the build files do not yet reflect the change.

**Comment by Mugen87** [comment]
Updated the builds now so it's easier to see Travis and lgtm errors.

File: examples/webgl_geometry_cube.html [hunk 1]
```diff
@@ -33,6 +33,8 @@
 				mesh = new THREE.Mesh( geometry, material );
 				scene.add( mesh );
 
+				console.log( THREE.MathUtils.clamp( 0.6, 0, 1 ) );
```
Comment (mrdoob) [inline examples/webgl_geometry_cube.html]: Leftover?

Comment (Mugen87) [reply examples/webgl_geometry_cube.html]: Ops^^. Let me remove this line...

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**